src/api/index.js

```diff
@@ -3,12 +3,16 @@ import { ENDPOINT } from "../constant";
 
 const API = axios.create({ baseURL: ENDPOINT });
 
-export const conversationIdFetch = async (senderId,receiverId ) => {
-  const conversation = await API.get(
-    `/getallconversationofuser/${senderId}/${receiverId}`
-  );
-  const fetchedConversation = conversation.data;
-  return fetchedConversation;
+export const conversationIdFetch = async (senderId, receiverId) => {
+  try {
+    const conversation = await API.get(
+      `/getallconversationofuser/${senderId}/${receiverId}`
+    );
+    const fetchedConversation = conversation.data;
```

Comment:
No need to declare the fetchedConversation, just return directly "return conversation.data"